tests/NHSD.BuyingCatalogue.Testing.Data/Entities/CatalogueItemEntity.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+namespace NHSD.BuyingCatalogue.Testing.Data.Entities
+{
+    public sealed class CatalogueItemEntity : EntityBase
+    {
+        public string CatalogueItemId { get; set; }
+
+        public int CatalogueItemTypeId { get; set; }
+
+        public DateTime Created { get; set; }
+
+        public string Name { get; set; }
+
+        public int PublishedStatusId { get; set; }
+
+        public string SupplierId { get; set; }
+
+        protected override string InsertSql => @"
+        INSERT INTO dbo.CatalogueItem
+        (
+            CatalogueItemId,
+            Name,
```

Comment:
Should `Name` include square brackets `[Name]`?

Reply:
It's not technically necessary, but we tend to include them to avoid it being highlighted as it's a non-reserved keyword (if I remember correctly) so we should include the square brackets here to be consistent.